kickoff/jsonexportl10n.py

```diff
@@ -15,37 +15,66 @@
 from jsonexport import myjsonify
 
 
+BETA_AGGREGATION_KEYWORD = "beta"
+
+
 @app.route('/json/l10n/<releaseName>.json', methods=['GET'])
 def l10nExport(releaseName):
     # Export the l10n changeset for a product
     releaseTable = getReleaseTable(releaseName)
+
+    if releaseName.endswith(BETA_AGGREGATION_KEYWORD):
+        l10n_list = _aggregateBetaLocales(releaseTable, releaseName)
```

Comment:
Discussed offline. The intent here is to show that function is private to the package. @sylvestre and I agreed on a follow up pathc to make the rest of the code base compliant to this rule.